enhancements/windows-containers/health-management.md

```diff
@@ -165,6 +181,11 @@ When making use of a PowerShell script, retries should be present within the scr
 is waiting for the HNS network to be created in order to create an HNS endpoint. In that specific case, the PowerShell
 script which creates the endpoint should include a retry period which waits for the needed Network to exist.
 
+Services marked with `bootstrap: true` will start before any others, using only their subset of priorities to determine
+order. Priorities extend beyond the `bootstrap` stage, meaning that in the `controller` stage, WICD will use priorities 
+of the whole service list to build its ordered list of services. This means that although a bootstrap service 
+starts before a non-bootstrap service, the non-bootstrap service can have a greater priority when WICD is reconciling.
```

Comment:
>This means that although a bootstrap service starts before a non-bootstrap service, the non-bootstrap service can have a greater priority when WICD is reconciling.

This does not compute for me. Please provide a scenario when there will be a controller only service that will have a higher priority than a bootstrap service.

You should also call out if a bootstrap service can depend on a controller service. PS: this is not possible.